libraries/AP_VideoTX/AP_VideoTX.cpp

```diff
@@ -75,6 +75,8 @@ const AP_Param::GroupInfo AP_VideoTX::var_info[] = {
 
 extern const AP_HAL::HAL& hal;
 
+const char * AP_VideoTX::band_names[] = {"A","B","E","F","R","L"};
```

Comment:
This is nice! Why Not use the proper names?
A, B, E, Airwave, Race, LoRace?

Reply:
We can do it, but again it's to keep text short and save flash

Reply:
Ok so if its too long I think you definitely need to distinguish between A,B,E and the others, most docs I see refer to Airwave rather than Fatshark, so F is quite misleading. How about "A, B, E, AW, RC, LO"?